lib/OpenQA/Worker.pm

```diff
@@ -287,11 +287,15 @@ sub init ($self) {
     Mojo::IOLoop->singleton->reactor->on(
         error => sub {
```

Comment:
this was missed

Reply:
With the signatures change I was focusing on `Job.pm`.

Reply:
I covered this function now nevertheless.